### PR TITLE
delete getting-started-ipynb

### PR DESCRIPTION
**Pull request by JustinaPetr**
This is a very outdated file and not necessary since we have started pack.

**Proposed changes**:
- delete the outdated getting-started.ipynb
- add a link to the starter pack
- add a few minor punctuation fixes in the readme

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

**Comment by MetcalfeTom** [comment]
@JustinaPetr are the punctuation fixes ready to push too?

**Comment by JustinaPetr** [comment]
@MetcalfeTom yes, all the proposed details are ready

**Comment by MetcalfeTom** [comment]
OK!  Feel free to add them here and I'll give them a quick review (currently this PR only adds the starter pack link to the readme)

File: README.md [hunk 1]
```diff
@@ -36,7 +39,7 @@ Chat Bots on :
 - Telegram
 - Twilio
 
-But you can also build assistants as
+But you can also build assistants as:
```
Comment (MetcalfeTom) [inline README.md]:
```suggestion
But you can also build assistants using:
```

**Pull request merged**